plugins/source/azure/resources/services/costmanagement/view_queries.go

```diff
@@ -0,0 +1,58 @@
+// Code generated by codegen2; DO NOT EDIT.
+package costmanagement
+
+import (
+	"github.com/cloudquery/plugin-sdk/schema"
+)
+
+func view_queries() *schema.Table {
+	return &schema.Table{
+		Name:     "azure_costmanagement_view_queries",
+		Resolver: fetchViewQueries,
+		Columns: []schema.Column{
+			{
+				Name:     "properties",
+				Type:     schema.TypeJSON,
+				Resolver: schema.PathResolver("Properties"),
+			},
+			{
+				Name:     "e_tag",
```

Comment:
This `ETag` to `e_tag` is another issue with the current codegen, I'm not addressing it in this PR.